apps/api/src/app/subscribers/subscribers.module.ts

```diff
@@ -5,11 +5,13 @@ import { USE_CASES } from './usecases';
 import { SubscribersController } from './subscribers.controller';
 import { AuthModule } from '../auth/auth.module';
 import { WidgetsModule } from '../widgets/widgets.module';
+import { GetPreferences, UpsertPreferences } from '@novu/application-generic';
+import { PreferencesRepository } from '@novu/dal';
 
 @Module({
   imports: [SharedModule, AuthModule, TerminusModule, forwardRef(() => WidgetsModule)],
   controllers: [SubscribersController],
-  providers: [...USE_CASES],
+  providers: [...USE_CASES, GetPreferences, UpsertPreferences, PreferencesRepository],
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** Add to `USE_CASES` instead? If not, please comment in code why not‏